modules/claims_api/spec/models/veteran_spec.rb

```diff
@@ -20,14 +20,6 @@
     end
   end
 
-  describe 'setting target veteran by OAuth' do
-    it 'instantiates from the OAuth user' do
-      identity = FactoryBot.create(:openid_user, identity_attrs: build(:user_identity_attrs, :loa3))
-      veteran = ClaimsApi::Veteran.from_identity(identity:)
-      expect(veteran.first_name).to eq(identity.first_name)
-    end
-  end
-
```

Comment:
Is the [from_identity method](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/models/claims_api/veteran.rb#L94) being tested anywhere now that this test is deleted?

Reply:
It is not being tested, but I don't see that the "from_identity" is being used anywhere outside of the file. Don't think it needs an rspec because of this

Reply:
Claims was refactored to use this module for getting the target veteran from an icn.
 - https://github.com/department-of-veterans-affairs/vets-api/blob/7a17e550a4ed301ad4294a7f7add008267f7ff9a/modules/claims_api/app/controllers/concerns/claims_api/target_veteran.rb#L10

This should be removed.

Reply:
> This should be removed.

@crolarlibertyva, what's "this" in that context?

Reply:
The entire rspec.

Reply:
I was affirming the removal of lines 23-30.

Reply:
ok, and I assume the TargetVeteran code is being tested somewhere? Can you link where it's being tested?

Reply:
Looks like it is being testing in multiple specs. For example:

- https://github.com/department-of-veterans-affairs/vets-api/blob/36d10215f0e81ee450cfdb12ce50e287fee27b52/modules/claims_api/spec/requests/v1/claims_request_spec.rb#L20
- https://github.com/department-of-veterans-affairs/vets-api/blob/36d10215f0e81ee450cfdb12ce50e287fee27b52/modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb#L3699

Reply:
👍